arches/app/models/migrations/0001_initial.py

```diff
@@ -112,6 +112,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ("contenttypes", "0002_remove_content_type_name"),
+        ("auth", "0002_alter_permission_name_max_length"),
```

Comment:
This dependency is necessary because this migration itself calls create_permission()